astropy/io/votable/tests/test_ucd.py

```diff
@@ -69,3 +69,57 @@ def test_invalid_namespace():
 def test_invalid_word():
     with pytest.raises(ValueError):
         ucd.parse_ucd("-pho")
+
+
+def test_atmospheric_wind_ucd():
+    """Test parsing of atmospheric observation UCDs, specifically obs.atmos.wind.
+
+    This test addresses issue #18452 where obs.atmos.wind and other new
+    atmospheric terms from UCD1+ version 1.6 should be recognized.
+    """
+    # Test the specific case from the issue report
+    ucd_string = "phys.veloc;obs.atmos.wind;stat.mean"
+    result = ucd.parse_ucd(
+        ucd_string, check_controlled_vocabulary=True, has_colon=";" in ucd_string
```

Comment:
I think this and subsequent uses of `has_colon` are not necessary here.  They are actually misleading in that the value looks like it will be `True` when there is a semicolon as opposed to a colon.  The `has_colon` parameter is in place only to support possible old values which contained colons in the actual vocabulary words.